.travis.yml

```diff
@@ -0,0 +1,18 @@
+dist: xenial
+sudo: false
+language: python
+cache: pip
+
+script:
+  - tox
+
+install:
+  - pip install tox tox-travis
+
+matrix:
+  include:
+    - python: "2.7"
+    - python: "3.4"
+    - python: "3.5"
+    - python: "3.6"
+    - python: "3.7"
```

Comment:
Instead of the `matrix:` block here you could simply add a `python:` block to the top of the file.

Ref. [Travis CI docs](https://docs.travis-ci.com/user/languages/python/)

Reply:
Nice, thank you!